run.py

```diff
@@ -165,7 +167,7 @@ def nonbib_to_master_pipeline(nonbib_engine, schema, batch_size=1):
     for current_row in session.query(models.NonBibTable).yield_per(100):
```

Comment:
the removed fields are large, so we should instead not load them in the first place, ie:

```
for current_row in session.query(models.NonBibTable).load_only('bibcode', .....).yield_per(100):
   ....
```

and then maybe call remove (if necessary) on what remains

Reply:
updated PR to only read database columns sent to master pipeline